$MirOS: ports/net/curl/patches/patch-lib_file_c,v 1.1 2008/04/05 21:54:39 tg Exp $
--- lib/file.c.orig	Thu May 22 17:37:13 2008
+++ lib/file.c	Tue Aug 26 19:03:21 2008
@@ -485,7 +485,7 @@ static CURLcode file_do(struct connectda
                Curl_wkday[tm->tm_wday?tm->tm_wday-1:6],
                tm->tm_mday,
                Curl_month[tm->tm_mon],
-               tm->tm_year + 1900,
+               (int)(tm->tm_year + 1900),
                tm->tm_hour,
                tm->tm_min,
                tm->tm_sec);
